test/proto/BUILD

```diff
@@ -11,6 +11,20 @@ envoy_package()
 
 exports_files(["bookstore.proto"])
 
+# The binary proto descriptors are generated and stored prior to runtime. The command to generate
+# proto descriptors from the command line can be found here: https://lyft.github.io/envoy/docs/conf
+# iguration/http_filters/grpc_json_transcoder_filter.html#config-http-filters-grpc-json-transcoder.
+# Alternatively, the proto descriptors can be generated with bazel rule envoy_proto_descriptor. The
```

Comment:
OK, I misread what you wrote, I thought you were saying the CLI arguement is "bazel rule envoy_proto_descriptor".

I think what I (and the person updating the bookstore example) want is just the verbatim command line you used to generate this binary descriptor, so that they can do the same. That would be more helpful than just pointing at the docs or feasible ways to write rules.